src/components/Register.jsx

```diff
@@ -101,7 +110,7 @@ function Register({ credentials, setCredentials }) {
 
       if (response === null) {
         window.alert(
-          "Registration failed.\n\nThe server could not be accessed.  Please try again later."
```

Comment:
I have made all the changes you have requested and I think it is good enough for an MVP. Please approve and merge this PR so that we can continue working on more features.